src/login/adal/AdalAuthProvider.ts

```diff
@@ -40,10 +34,10 @@ export class AdalAuthProvider extends AuthProviderBase {
 			// eslint-disable-next-line @typescript-eslint/no-explicit-any
 			log: (_level: any, message: any, error: any) => {
 				if (message) {
-					super.outputChannel.appendLine(message);
+					this.outputChannel.appendLine(message);
```

Comment:
Apparently `outputChannel` isn't accessible on `super` even though it compiles  🤷‍♂️